### PR TITLE
Improve npm packages' metadata

### LINKED ISSUE #5388: Add `repository` to addons' `package.json`
Follow-up for  #5243: The react-addons-* packages should have a `repository` entry in their `package.json` files.

### PR DESCRIPTION
**Pull request by zpao**
fixes #5388

But also makes the readmes for the addons packages better. npm pulls the first paragraph as a subheader which looks like shit.

screen shot 2015-11-04 at 5 34 15 pm

This moves the link sentence to a standalone line so it shouldn't be part of that text and also gives each addon a link to its actual docs (except for shallowCompare which doesn't have docs… see #5378)

### CODE REVIEW

File: grunt/tasks/npm-react-addons.js [hunk 1]
```diff
@@ -76,15 +85,20 @@ function buildReleases() {
...
       path.join(destDir, 'README.md'),
-      '# ' + pkgName + '\n\n' +
-      'This package provides the React ' + k + ' add-on. See ' +
-      'http://facebook.github.io/react/docs/addons.html for more information.\n'
+      `
+# ${pkgName}
+
+This package provides the React ${k} add-on.
+
+See <${link}> for more information.`.substr(1)
```
Comment (spicyj) [inline grunt/tasks/npm-react-addons.js]: nit: `slice` is easier to understand and you can't mix it up with `substring`. I hate substr/substring. You can also ignore this.

File: packages/react-addons/package.json [hunk 2]
```diff
@@ -2,6 +2,11 @@
   "name": "react-addons",
```
Comment (spicyj) [inline packages/react-addons/package.json]: nit: can we change this to `react-addon-template` or something? I always think it's a real thing.

**Comment by spicyj** [comment]
:+1:

File: packages/react-addons/package.json (hunk 2 above)
Comment (zpao) [inline packages/react-addons/package.json]: Good idea, will do.

File: grunt/tasks/npm-react-addons.js (hunk 1 above)
Comment (zpao) [inline grunt/tasks/npm-react-addons.js]: 3 calls that all do the same thing… yay JS!

**Pull request merged**